_posts/2024/2024-02-06-our-process.md

```diff
@@ -0,0 +1,453 @@
+    1.  company goals
+    2.  users' needs
+    3.  technical opportunities
+-   Key Questions:
+    -   What should each engineering team focus on and why?
+-   Frequency: quarterly
+-   Responsible: VP Engineering
+-   Contributors: entire engineering team
+-   Products:
+    -   Prioritized list of objectives and key results
+
+> Note: we used to define more of these each quarter than we do now.
+> The theory was that if we're meeting all of our targets,
+> our targets weren't ambitious enough.
+> We now realize that setting ourselves up for continual failure like that isn't actually helpful.
+
+### Engineering OKR Reviews
+
+Improvement in anything requires [reflective practice][reflection].
+Our end-of-quarter OKR reviews therefore have two purposes:
+to make sure everyone understands where we actually are,
+and to summarize what we've learned so that we can do better next time.
+
+-   Summary: End of quarter sync on OKR progress
+-   Key Questions:
+    -   What is the status of each OKR?
+    -   What progress did each team make towards overall goals?
+    -   What losses, challenges, and obstacles did we encounter?
+    -   What did we learn?
+-   Frequency: quarterly
+-   Responsible: VP Engineering
+-   Contributors: project leads and other team members
+-   Products:
+    -   Ranked list of things that worked well and things that didn't
+    -   Ranked list of ideas for things to try to adopt, improve, or drop
+
+### Engineering Quarterly Retrospective
+
+Similarly,
+we have a general retrospective at the end of each quarter
+to ask ourselves wider-ranging questions
+that may not be specific to a particular project's goals.
+Practices like having an on-call user support rotation
+have grown out of these self-reviews.
+
+-   Summary: End of quarter team sync to reflect on the quarter and identify ways to improve how we work and our ability to deliver
+-   Key Questions:
+    -   What went well?
+    -   What didn't?
+    -   What changes would we like to implement to improve the way we work?
+-   Frequency: quarterly
+-   Responsible: Scrum master
+-   Contributors: entire engineering team
+-   Products:
+    -   Ranked list of things that worked well and things that didn't
+    -   Ranked list of ideas for things to try to adopt, improve, or drop
+
+> Note: our product manager has been doubling as our scrum master for the last couple of years.
+> Those jobs are usually done by different people in larger organizations,
+> so the description above lists the role rather than the person who happens to be doing it right now.
+
+### Engineering All Hands
+
+Our company includes biologists working in a wet lab,
+data scientists,
+and machine learning specialists staring into the void and hoping it blinks first.
+And while most of us are in Toronto,
+a substantial minority live and/or work remotely.
+This meeting is where the VP of Engineering updates us all
+on things we might otherwise miss
+(both organizationally and technically).
```

Comment:
is it worth calling out other ways we use this meeting? Yes, Asha gives updates, but we also use it for guest speakers, AMAs with leadership, deep dives on particular topics of interest, or tools we're working on. There might be merit in calling it out as these are also valuable uses of this time.